metrics/ci_integrations/docs/features/supported_storage_version/diagrams/ci_integrations_supported_storage_version_class_diagram.puml

```diff
@@ -0,0 +1,116 @@
+@startuml
```

Comment:
Would it be better to name this file something like `supported_storage_version_class_diagram.puml`?